src/utils/date.js

```diff
@@ -55,6 +55,22 @@ export function isValid (date) {
   return isNaN(t) === false
 }
 
+export function isInDatesArray (datesArray, date) {
+  let boolVal = false
+  if (datesArray === undefined) {
+    return boolVal
+  }
+  else {
+    for (var i = 0; i < datesArray.length; i++) {
+      if (datesArray[i].toDateString() === date.toDateString()) {
```

Comment:
There must be a more elegant way of doing this, because what if there are hundreds of thousands (or millions) of days? Wouldn't it be faster to convert the dates to unix time and simply check if it is >== and <== ???

Reply:
I was wondering if we should limit the range here. We could assume the client dev won't be using 1000's of dates in the array, because that would be kind of silly for the users to be selecting from. I dunno....

Scott

Reply:
Well, what if they have a range of dates that could be anywhere from 15000 BCE until 2020 CE? (Like if they are building a timeline for the development of artificial intelligence...) There are more than a couple thousand days there...

Reply:
I've added a limit to 1000 items (dates). 

Scott